src/app-sync.js

```diff
@@ -314,6 +321,13 @@ app.get('/list-user-files', (req, res) => {
       groupId: row.groupId,
       name: row.name,
       encryptKeyId: row.encryptKeyId,
+      owner: row.owner,
+      usersWithAccess: fileService
+        .findUsersWithAccess(row.id)
+        .map((access) => ({
+          ...access,
+          owner: access.userId === row.owner,
+        })),
```

Comment:
It may not be for this PR, but this seems a little inefficient to me. I think it would be better to have `findUsersWithAccess(rows.map(({id}) => id))` so we don't have to look at disk for each separate file. What do you think?

(same comment below for `/get-user-file-info`)

Reply:
Yeah, it makes sense. I'll make a follow-up issue to track this as an optimization